src/profile-logic/processed-profile-versioning.js

```diff
@@ -527,5 +527,69 @@ const _upgraders = {
       }
     }
   },
+  [12]: profile => {
+    // Modern categories
+    profile.meta.categories = [
+      {
+        name: 'Idle',
+        color: 'transparent',
+      },
+      {
+        name: 'Other',
+        color: 'grey',
+      },
+      {
+        name: 'JavaScript',
+        color: 'yellow',
+      },
+      {
+        name: 'Layout',
+        color: 'purple',
+      },
+      {
+        name: 'Graphics',
+        color: 'green',
+      },
+      {
+        name: 'DOM',
+        color: 'blue',
+      },
+      {
+        name: 'GC / CC',
+        color: 'orange',
+      },
+      {
+        name: 'Network',
+        color: 'lightblue',
+      },
+      {
+        name: 'HTML Parsing',
+        color: 'brown',
+      },
+    ];
+    const oldCategoryToNewCategory = {
+      [1 << 4 /* OTHER */]: 1 /* Other */,
+      [1 << 5 /* CSS */]: 3 /* Layout */,
+      [1 << 6 /* JS */]: 2 /* JavaScript */,
+      [1 << 7 /* GC */]: 6 /* GC / CC */,
+      [1 << 8 /* CC */]: 6 /* GC / CC */,
+      [1 << 9 /* NETWORK */]: 7 /* Network */,
+      [1 << 10 /* GRAPHICS */]: 4 /* Graphics */,
+      [1 << 11 /* STORAGE */]: 1 /* Other */,
+      [1 << 12 /* EVENTS */]: 1 /* Other */,
+    };
```

Comment:
Is it possible to share these 2 objects between this file and the gecko-profile-versioning.js ? Maybe it's not that important given it won't be used elsewhere, so I'll let you decide.

Reply:
I don't think it's worth sharing them.